bin/release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+kob_version="$1"
+
+branch="REL-taggin"
+
+kob_archive_download_repo="KOBman_target_repo"
+
+kob_namespace= ${KOB_NAMESPACE:-hyperledgerkochi}
+
+#sanityls
+if [[ -z "$VERSION" ]]; 
+    then
+        echo "Usage: release.sh <version>"
+ 	    exit 0
+fi
+
+echo "$kob_version"
+#prepare branch
+# echo "prepare branch"
+# #git fetch
+git checkout Dev
```

Comment:
dev should be Small caps

Reply:
This Dev is the name of my remote branch. I have been working from it